projects/storyteller/webapp/proto/data.proto

```diff
@@ -70,7 +70,7 @@ message StoryItem {
 
 message Story {
   int64 start_time_ms = 1;
-  int64 endTimeMs = 2;
+  int64 end_time_ms = 2;
```

Comment:
Great! Thanks! But as I said, when you fixed that, your proto functions is changed too.
Probably you have outdated proto functions, that's why you didn't get an error.
But after each proto update, you need to generate new proto functions too:
https://github.com/hasadna/hasadna/tree/master/projects/storyteller/webapp#updating-protos

So if you'd do that, you're going to see the error:
https://i.imgur.com/yqchTXD.png

To avoid it, you need to find proto variables with invalid case and fix it.
Example:
https://github.com/hasadna/hasadna/blob/addingAngularMaterial/projects/storyteller/webapp/src/app/components/feed/feed.component.html#L14